aten/src/ATen/native/cuda/int4mm.cu

```diff
@@ -0,0 +1,1020 @@
+#if (defined(CUDA_VERSION) && CUDA_VERSION >= 12000) && (!defined(__CUDA_ARCH__) || (__CUDA_ARCH__ >= 800))
+#include <cuda_bf16.h>
+#include <cuda_fp16.h>
+#include <cuda_runtime.h>
+#include <mma.h>
+#endif
+#include <ATen/ATen.h>
+#include <ATen/core/Tensor.h>
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/DeviceGuard.h>
+#include <c10/cuda/CUDAGuard.h>
+#include <torch/types.h>
```

Comment:
you need to include something like `#include <ATen/native/cuda/_weight_int4pack_mm_impl.cuh>` and `_convert_weight_to_int4pack.cuh` or something like that

Reply:
```_weight_int4pack_mm``` and ```_convert_weight_to_int4pack``` are the two functions I added in this file, I don't have these two ```cuh``` file.

Reply:
Should I make two fake ```cuh``` file to make CI happy?